consensus/core/src/error.rs

```diff
@@ -13,7 +14,29 @@ pub enum ConsensusError {
 
     #[error("RocksDB failure: {0}")]
     RocksDBFailure(#[from] TypedStoreError),
+
+    #[error("FastCrypto failure: {0}")]
+    CryptographicOperationFailure(#[from] FastCryptoError),
```

Comment:
Should we make this a specific error for us that it was either a signer error or a signature verification error?